src/main/scala/xebia/cakemix/AskTimeoutProvider.scala

```diff
@@ -17,7 +17,9 @@ trait AskTimeoutProvider {
 }
 
 object AskTimeoutProvider {
-  def fromConfig(config: Config, key: String): Timeout = {
+  private val AskTimeoutConfigKey = "xebia.cakemix.ask-timeout"
```

Comment:
But still why would you want to lose that visibility for users of your library? It adds to findability of `xebia` if you'd keep it in there.

At the risk of sounding processy: Are there some Xebia guidelines on releasing this kind of cool open source libraries? They could potentially be nice marketing/recruitment tools ;)